foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -382,9 +382,11 @@ This might seem like a lot of code if you're attaching lots of similar event lis
 
 ```HTML
 <div id="container">
+
   <button id="1">Click Me</button>
   <button id="2">Click Me</button>
   <button id="3">Click Me</button>
+
 </div>
 ```
```

Comment:
This code block doesn't need new lines inserted. It does however need the language to be in lower case for style consistency.
````suggestion
```html
<div id="container">
  <button id="1">Click Me</button>
  <button id="2">Click Me</button>
  <button id="3">Click Me</button>
</div>
```
````